.github/workflows/conda.yml

```diff
@@ -0,0 +1,116 @@
+name: Conda
+
+on:
+  pull_request:
+  release:
+  prerelease:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test on ${{ matrix.cfg.os }}, Python ${{ matrix.python-version }}, OpenEye=${{ matrix.openeye }}
+    runs-on: ${{ matrix.cfg.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        openeye:
+          - "true"
+          - "false"
+        cfg:
+          - os: ubuntu-latest
+          - os: macOS-latest
+
+    env:
+      CI_OS: ${{ matrix.cfg.os }}
+      OPENEYE: ${{ matrix.cfg.openeye }}
+      PYVER: ${{ matrix.python-version }}
+      OE_LICENSE: ${{ github.workspace }}/oe_license.txt
+      PACKAGE: openforcefield
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Wait for 2 hours if triggered by a release
+        if: (github.event_name == 'release') || (github.event_name == 'prerelease')
+        run:
+            sleep 7200
```

Comment:
This wait time is conservative but I'd like to maximize the chance that a release actually is up on Anaconda Cloud (build time + ~1  hour for their CDN to update + a larger buffer)

It should not run into walltime limits (6h and 72h limits for single jobs & workflows): https://docs.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits

Reply:
I could take this or leave it. It's a nice concept in theory, but will likely spell confusion in the case of a rolled-back or held-up release ("we rolled everything back an hour ago -- what's this failing test? Is something really wrong?")

Reply:
Good point - if we only have this running as a ~nightly job, we'd still be able to catch breaking changes in dependencies but it would not interfere with the release process.

Reply:
Hm, I think this merging this PR has a lot of value even without figuring out exactly how it will slot into our release process. So I'd push to merge without resolving this question.

But, that said, I think our release process should probably unconditionally push a package to `omnia/label/rc`, and the release tests should target the package built on that label. So I _do_ think there should be release automation, but it should be a different workflow, which may or may not use this file.